src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import './style.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
@Ochowo. Instead of doing this. I think It's better to add the eslint rule in the .eslintrc file. That way we won't have eslint rules across all our files. As we will be adding more as time goes on.
```
rules: {
 'react/prefer-stateless-function': 0,
}
```

Reply:
@Tyak99 I don't think we should disable this rule. A better option will be to use a function for stateless components and a class for components with state.